test/tools/d_do_test.d

```diff
@@ -596,24 +600,52 @@ bool compareOutput(string output, string refoutput, const ref EnvData envData)
             import std.ascii : isDigit;
             output.skipOver!isDigit();
         }
-        else // $p:<some path>$
+        else // $<identifier>:<special content>$
         {
```

Comment:
You could reduce the indenting by getting rid of the else branch by putting a continue after `output.skipOver!isDigit();`

Reply:
Done